src/shared/sleep-config.c

```diff
@@ -46,6 +50,7 @@
 #define DISCHARGE_RATE_FILEPATH "/var/lib/systemd/sleep/battery_discharge_percentage_rate_per_hour"
 #define BATTERY_DISCHARGE_RATE_HASH_KEY SD_ID128_MAKE(5f,9a,20,18,38,76,46,07,8d,36,58,0b,bb,c4,e0,63)
 #define SYS_ENTRY_RAW_FILE_TYPE1 "/sys/firmware/dmi/entries/1-0/raw"
+#define SPECIAL_USER_SLICE "user.slice"
```

Comment:
SPECIAL_USER_SLICE is already declared exactly like that in `special.h`

Reply:
fixed